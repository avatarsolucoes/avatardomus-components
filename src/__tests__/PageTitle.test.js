import React from 'react';
// import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import { PageTitle } from '..';

describe('PageTitle', () => {
  it('is truthy', () => {
    expect(PageTitle).toBeTruthy();
  });

  it('Test renderer whith children', () => {
    const estilo = {
      width: '100%',
      maxWidth: '100%',
    };

    const wrapper = shallow(
      <PageTitle className="test" style={estilo}>
        <h1>Test Title</h1>
      </PageTitle>,
    );
    expect(wrapper.exists('.test')).toBe(true);
    expect(wrapper.exists('h1')).toBe(true);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
