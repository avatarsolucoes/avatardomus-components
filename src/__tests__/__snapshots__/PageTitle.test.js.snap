// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTitle Test renderer whith children 1`] = `"<div class=\\"page-title test\\" style=\\"width:100%;max-width:100%\\"><h1>Test Title</h1></div>"`;
